fvtest/omrGtestGlue/omrTest.h

```diff
@@ -37,17 +37,9 @@ static int iconv_init_static_variable = iconv_initialization();
  * So we explicitly include <ctype.h> and undefine the macros for gtest, after gtest we then define back the macros.
  */
```

Comment:
This include of `ctype.h` should also be unconditional, and the stale comment (lines 35-38) removed.